Ginger/Ginger/UserControlsLib/ucGridView/ucGrid.xaml.cs

```diff
@@ -2137,7 +2140,10 @@ public ObservableList<RepositoryItemBase> GetSelectedItems()
             ObservableList<RepositoryItemBase> selectedItemsList = new ObservableList<RepositoryItemBase>();
             foreach (object selectedItem in grdMain.SelectedItems)
             {
-                selectedItemsList.Add((RepositoryItemBase)selectedItem);
+                if (selectedItem is RepositoryItemBase)
```

Comment:
you added the same check also to ucListView?

Reply:
Yes !